assets/js/components/user-input/UserInputPreview.js

```diff
@@ -79,80 +58,90 @@ export default function UserInputPreview( { noFooter, back, goTo, redirectURL }
 		USER_INPUT_ANSWERS_ROLE,
 	} = getUserInputAnwsers();
 
+	const { error } = useSelect( ( select ) => ( {
+		error: select( CORE_USER ).getErrorForAction( 'saveUserInputSettings', [] ),
+	} ) );
```

Comment:
@benbowler :man_facepalming: excuse me for the incorrect suggestion at the beginning, but we need to move error handling back to the `UserInputQuestionnaire` component and render it there instead of keeping it in the preview component. The idea here is that we need to display error no matter which screens we are on (a single question or the preview).

Reply:
Are you happy with the error message displaying here outside the questions box? If we want it inside the box we will need to pass it as a prop to `UserInputQuestionWrapper` instead.

<img width="1096" alt="Screenshot 2021-01-05 at 16 59 23" src="https://user-images.githubusercontent.com/119352/103675143-7e15ea00-4f77-11eb-89de-79853f105e88.png">

Reply:
I've inlined this now so that it looks like the following:

<img width="1113" alt="Screenshot 2021-01-06 at 10 38 32" src="https://user-images.githubusercontent.com/119352/103760219-b3b8e280-500c-11eb-86af-16a56916e89e.png">
<img width="1100" alt="Screenshot 2021-01-06 at 10 39 21" src="https://user-images.githubusercontent.com/119352/103760231-b7e50000-500c-11eb-8e60-a40d4065e594.png">